compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmSharedVariablesManager.kt

```diff
@@ -75,7 +75,7 @@ class JvmSharedVariablesManager(
         RefProvider(refClass, refClass.typeParameters[0].defaultType)
     }
 
-    private fun getProvider(valueType: IrType): RefProvider =
+    fun getProvider(valueType: IrType): RefProvider =
```

Comment:
Please suppress the "function could be private" inspection and add the same annoying comment that it's used in the plugin. :)

Reply:
Fixed. I couldn't guess the comment, but added a pointer to the use-site. Again, these are not showing up in my build, unfortunately, because it _is_ seeing the use in FragmentSharedVariablesLowering.